runbot_send_email/models/runbot_build.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+# © 2016 Vauxoo
+#   Coded by: lescobar@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+from travis2docker.git_run import GitRun
+from travis2docker.travis2docker import main as t2d
+
+import openerp
+from openerp import models
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotBuild(models.Model):
+    _inherit = 'runbot.build'
+
+    def send_email(self, cr, uid, ids, context=None):
+        context = context or {}
+        build_data = self.read(cr, uid, ids, ['dest', 'state',
+                                              'committer_email',
+                                              'committer'])[0]
```

Comment:
@lescobarvx use a `for build_data in self.read(cr, uid, ids, ['dest', 'state', 'committer_email', 'committer']):` to get all ids